CMakeLists.txt

```diff
@@ -36,13 +36,13 @@ endif()
 ## =================================================================================================
 ## Test and Benchmarks target
 ## =================================================================================================
-include(CTest)
-include(target/add_parent_target)
-include(config/compiler)
-include(config/types)
-include(config/pch)
-
 if( EVE_BUILD_TEST )
+  include(CTest)
+  include(target/add_parent_target)
+  include(config/compiler)
+  include(config/types)
+  include(config/pch)
```

Comment:
This file which is included here adds "documentation" related executable while the if above will make sure that executable won't be created.

Meaning that I think we're breaking things for someone who disables "tests" and enables "documentation" here.

Reply:
As said in the issue, this documentation source thing is gonna be removed by the doc PR anyway so only test and bench options will remain.

Reply:
See https://github.com/jfalcou/eve/pull/1124/files?file-filters%5B%5D=.cmake&hide-deleted-files=true

Reply:
Oh, that's okay then. thanks